tests/test_events.py

```diff
@@ -12,7 +12,7 @@
 from website.addons.base import signals
 from framework.auth import Auth
 from tests import factories
-from tests.base import OsfTestCase
+from tests.base import OsfTestCase, NotificationTestCase
```

Comment:
Why switch to `NotificationTestCase` in this file? I don't see any test behavior change accompanying this base class switch.